src/main/java/com/on_site/q/Q.java

```diff
@@ -647,7 +649,7 @@ private int adjustIndex(int index) {
     private static final LoadingCache<Document, Frizzle> FRIZZLES = CacheBuilder.newBuilder().weakKeys().softValues().build(new CacheLoader<Document, Frizzle>() {
```

Comment:
Should we drop the `softValues()` call? It seems unnecessary to me with these changes.